src/Libraries/DSOffice/Excel.cs

```diff
@@ -374,6 +376,13 @@ private static object[][] ConvertToJaggedArray(object[,] input)
                         {
                             output[i, j] = "";
                         }
+                        else if (item is StackValue)
+                        {
+                            if(((StackValue)item).IsPointer)
```

Comment:
Ideally, these solutions should refer to an API that exposes Engine/Mirror functionality.
This doesnt exist yet.
This is the task that should properly address this dependency on ProtoCore:
http://adsk-oss.myjetbrains.com/youtrack/issue/MAGN-7341